testing/state-transition/state-transition.go

```diff
@@ -90,12 +97,12 @@ func initTestStores() (*beacondb.KVStore, *depositstore.KVStore, error) {
 
 	cms.MountStoreWithDB(testStoreKey, storetypes.StoreTypeIAVL, nil)
 	if err = cms.LoadLatestVersion(); err != nil {
-		return nil, nil, fmt.Errorf("failed to load latest version: %w", err)
+		return nil, nil, nil, fmt.Errorf("failed to load latest version: %w", err)
 	}
 
-	ctx := sdk.NewContext(cms, true, nopLog)
-	testStoreService := &testKVStoreService{ctx: ctx}
-	return beacondb.New(testStoreService),
+	testStoreService := &testKVStoreService{}
+	return cms,
```

Comment:
we need to export the multistore in order to be able to create temporary cache states out of it, each for a single block.
This does not affect other tests